.github/workflows/ci.yml

```diff
@@ -161,10 +161,9 @@ jobs:
         uses: grafana/shared-workflows/actions/push-to-gar-docker@751820b271417d91fe8588816ed4296b311caa33 # v0.5.1
         with:
           # Only push to GAR on main branch pushes
-          push: ${{ github.event_name == 'push' && github.ref_name == 'main' }}
```

Comment:
I assume this will be reverted when this PR is ready for review?

Should this be a Draft for now?